File: android/app/src/main/java/com/vegettable/app/model/MarketPrice.java
package com.vegettable.app.model;

import com.google.gson.annotations.SerializedName;

public class MarketPrice {
    @SerializedName("marketName")
    private String marketName;

    @SerializedName("cropName")
    private String cropName;

    @SerializedName("avgPrice")
    private double avgPrice;

    @SerializedName("upperPrice")
    private double upperPrice;

    @SerializedName("lowerPrice")
    private double lowerPrice;

    @SerializedName("volume")
    private double volume;

    @SerializedName("transDate")
    private String transDate;

    public String getMarketName() { return marketName; }
    public String getCropName() { return cropName; }
    public double getAvgPrice() { return avgPrice; }
    public double getUpperPrice() { return upperPrice; }
    public double getLowerPrice() { return lowerPrice; }
    public double getVolume() { return volume; }
    public String getTransDate() { return transDate; }
}

File: android/app/src/main/java/com/vegettable/app/model/MarketPrice.kt
package com.vegettable.app.model

import com.google.gson.annotations.SerializedName

class MarketPrice {
    @JvmField
    @SerializedName("marketName")
    val marketName: String? = null

    @SerializedName("cropName")
    val cropName: String? = null

    @JvmField
    @SerializedName("avgPrice")
    val avgPrice: Double = 0.0

    @JvmField
    @SerializedName("upperPrice")
    val upperPrice: Double = 0.0

    @JvmField
    @SerializedName("lowerPrice")
    val lowerPrice: Double = 0.0

    @SerializedName("volume")
    val volume: Double = 0.0

    @JvmField
    @SerializedName("transDate")
    val transDate: String? = null
}
